### PR TITLE
feat: check if --version is a correct semver range

### PR DESCRIPTION
**Pull request by michalgolab**

### CODE REVIEW

File: internal/cmd/get/get.go [hunk 1]
```diff
@@ -43,6 +44,21 @@ func NewCommand(ctx context.CLIContext) *cobra.Command {
 	return cmd
 }
 
+func validateSemVer(versionString string) bool {
+	// Official RegEx from https://semver.org/spec/v2.0.0.html
+	const semVerRegexp = `^(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)` +
```
Comment (crazybarber) [inline internal/cmd/get/get.go:49]: Please change it to some out-of-the-box solution. E.g. https://github.com/Masterminds/semver

**Review by crazybarber — CHANGES_REQUESTED** [review]

File: internal/cmd/get/get.go [hunk 2]
```diff
@@ -43,6 +45,11 @@ func NewCommand(ctx context.CLIContext) *cobra.Command {
 	return cmd
 }
 
+func validateSemVer(versionString string) bool {
```
Comment (crazybarber) [inline internal/cmd/get/get.go:48]: I would simplify and remove this func. Then in 71 simply use `semver.STtrictVersion` with err handling.

**Review by crazybarber — APPROVED** [review]

**Review by crazybarber — APPROVED** [review]

**Review by savoc — APPROVED** [review]

File: internal/cmd/get/get.go [hunk 3]
```diff
@@ -61,6 +62,11 @@ func getCommand(ctx context.CLIContext, opts *options, args []string) {
 		log.Fatalf("scope initialization failed: %s", err)
 	}
 
+	_, err = semver.StrictNewVersion(opts.Version)
```
[the remark this answers is not in the document]
Comment (michalgolab) [reply internal/cmd/get/get.go]: I made the changes according to your suggestion.

File: internal/cmd/get/get.go [hunk 4]
```diff
@@ -61,6 +62,10 @@ func getCommand(ctx context.CLIContext, opts *options, args []string) {
 		log.Fatalf("scope initialization failed: %s", err)
 	}
 
+	if _, err := semver.StrictNewVersion(opts.Version); err != nil {
+		log.Fatalf("'%s' does not meet the criteria for semantic versioning", opts.Version)
```
Comment (savoc) [inline internal/cmd/get/get.go:66]: heh, I know I should've mentioned it before, but you could also add the error message returned by that semver check to your fatal log <3

**Review by savoc — APPROVED** [review]

File: internal/cmd/get/get.go (hunk 4 above)
Comment (michalgolab) [reply internal/cmd/get/get.go]: Done

**Review by crazybarber — APPROVED** [review]

**Review by crazybarber — APPROVED** [review]

**Pull request merged**